brave_domains/service_domains.h

```diff
@@ -9,8 +9,11 @@
 #include <string>
 
 #include "base/command_line.h"
+
 namespace brave_domains {
 
+enum ServicesEnvironment { kDev, kStaging, kProd };
```

Comment:
According to the chromium c++ style guide, these should be in the format `DEV`, `STAGING` and `PROD` (should it be `PRODUCTION`?)